### PR TITLE
[WIP] Expanding Auth0 Extend with Compilers

### PR DESCRIPTION
**Pull request by cfjedimaster**
Please review....

### CODE REVIEW

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Expanding Auth0 Extend with Compilers
+description: A look at how webtask compilers can open your extensions to new possibilities.
+longdescription: <A LONG DESCRIPTION OF THE POST BETWEEN 230 AND 320 CHARACTERS>
```
Comment (NotMyself) [inline _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Need to add a long description here.

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+category: Extend, Technical, Webtasks
+is_non-tech: false
+author:
+  name: Raymond Camden
+  url: https://www.raymondcamden.com
+  mail: jedimaster@auth0.com
+  avatar: "https://static.raymondcamden.com/images/ray3_2017.jpg"
+design:
+  bg_color: "#3445DC"
+  image: <A PATH TO A 200x200 IMAGE>
```
Comment (NotMyself) [inline _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Need to request an image from #mktg_assets_requst for this and a twitter card.

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+tags:
+- extend
+- webtasks
+- serverless
+related:
+- <ADD SOME RELATED POSTS FROM AUTH0'S BLOG>
```
Comment (NotMyself) [inline _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Try to add at least 3 related posts here.

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+---
+
+Compilers are easily the most powerful, and perhaps the hardest to grasp, capabilities of Webtask and Auth0 Extend. In a nutshell, compilers provide a way to completely modify, customize, and expand, how you (and if you are using Extend, your users) build serverless extensions. In this post I'm going to demonstrate how to use compilers and give you some ideas of how to use them with Auth0 Extend.
```
Comment (NotMyself) [inline _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: One of these Auth0 Extend references should probably be a link to auth0.com/extend.

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown (hunk 3 above)
Comment (cfjedimaster) [reply _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: i tried like heck but found one that was - barely - linkable i think - https://auth0.com/blog/introducing-auth0-hooks/. have any suggestions?

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown (hunk 1 above)
Comment (cfjedimaster) [reply _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Added.

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown (hunk 4 above)
Comment (cfjedimaster) [reply _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Added.

File: _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown (hunk 2 above)
Comment (brunokrebs) [reply _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Actually, Ramiro takes care of it when scheduling for publish

Comment (cfjedimaster) [reply _posts/2018-03-27-expanding-aoth0-extend-with-compilers.markdown]: Ok, I asked on the channel, but I'll let R know as well.

**Pull request merged**